lib/repl.js

```diff
@@ -320,9 +323,21 @@ function REPLServer(prompt,
           // value for statements and declarations that don't return a value.
           code = `'use strict'; void 0;\n${code}`;
         }
+        let pwd;
+        try {
+          const { pathToFileURL } = require('url');
+          pwd = pathToFileURL(process.cwd()).href;
+        } catch {
+        }
```

Comment:
Nit: please move it out of the loop, since it won't change inside of the loop.

Reply:
fixed